### PR TITLE
Further removal of test logging to StdOut/StdErr

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

File: test/src/org/exist/xmldb/ResourceTest.java [hunk 1]
```diff
@@ -254,7 +254,6 @@ public void queryRemoveResource() throws XMLDBException {
 
         for (ResourceIterator iterator = rs.getIterator(); iterator.hasMoreResources();) {
             Resource r = iterator.nextResource();
-            System.err.println("Resource id=" + r.getId() + " xml=" + r.getContent());
```
Comment (dizzzz) [inline test/src/org/exist/xmldb/ResourceTest.java]: no fail() replacement ?

Comment (adamretter) [reply test/src/org/exist/xmldb/ResourceTest.java]: It was not previously considered a fail. I suspect, it was previously meant to say `System.out.println`, but well both in tests apart from debugging is bad.

Comment (adamretter) [reply test/src/org/exist/xmldb/ResourceTest.java]: @dizzzz We good?

Comment (dizzzz) [reply test/src/org/exist/xmldb/ResourceTest.java]: yes, sorry.

**Pull request merged**